hack/patches/003-routeretry.patch

```diff
@@ -0,0 +1,51 @@
+diff --git a/vendor/knative.dev/pkg/test/spoof/openshift_checks.go b/vendor/knative.dev/pkg/test/spoof/openshift_checks.go
```

Comment:
This patch is required to make up for the dropped `EventuallyMatchesBody`, which was essentially a universal retry.